### ISSUE TITLE
Create Uploader Lambda and import Storage Bucket

### ISSUE THREAD

**Issue by ciaransweet** [NASA-IMPACT/hls-sentinel2-downloader-serverless#6]
# TL;DR

The Lambda responsible for reading files from the EFS volume and uploading them to the imported S3 bucket needs to be created

![image](https://user-images.githubusercontent.com/9111975/102630829-ede93f80-4144-11eb-8738-7fa5191968d5.png)

# Info

Currently the downloader checks to see if the uploader queue has items and then processes them. This happens every second, even if there might not be any images available to upload. 

The new approach will have the Lambda invoked as soon as there is a message in the To Upload Queue, as we're not limited to concurrent connections to S3 to upload, this will not have a concurrent execution limit. This creates a bottleneck on the downloader for the speed of link -> uploaded, but that shouldn't be a problem.

The pseudo code for the Uploader Lambda is:

```python
# Downloader Lambda downloads a file and populates the To Upload Queue, this then invokes our function:

def handler(event, context):
    filepath, id = get_filepath_and_db_id(event)
    receipt_handle = event['receiptHandle']
    with open(filepath, 'rb') as file_contents:
        s3.put_object(Bucket=STORAGE_BUCKET, Key=filename)
    update_db(id) # Set to uploaded
    confirm_message_received(receipt_handle)
    delete_file(filepath)
```

# Acceptance criteria:
* Lambda function and associated methods within are unit tested
* Lambda function is deployed and has access to the Storage Bucket
* EFS Volume is attached to the Lambda
* Integration test is run which ensures:
    * Function is invoked on the To Upload Queue having a message
    * The file is uploaded (Use a bucket deployed via an integration stack)
    * Ensure file is deleted off EFS (Deploy a lambda in the integration tests and assure that the file isn't present)
    * Check that granule table is updated correctly
